.changeset/smooth-goats-agree.md

```diff
@@ -0,0 +1,49 @@
+---
+'astro': minor
+---
+
+Add support for generating multiple widths when using the Image component and a Picture component for supporting multiple formats.
+
+## `srcset` support
+
+Two new properties have been added to `Image` and `getImage`: `densities` and `widths`.
+
+These props can be used to generate a `srcset` attribute with multiple sources. For example:
+
+```astro
+---
+import { Image } from "astro";
+import myImage from "./my-image.jpg";
+---
+
+<Image src={myImage} width={myImage.width / 2} densities={[2]} alt="My cool image"  />
+```
+
+```html
+<img src="..." srcset="... 2x, ... 3x" alt="My cool image" />
+```
+
+## Picture component
+
+The `Picture` component can be used to generate a `<picture>` element with multiple sources. It can be used as follow:
+
+```astro
+---
+import { Picture } from "astro:assets";
+import myImage from "./my-image.jpg";
+---
+
+<Picture src={myImage} formats={["avif", "webp"]} alt="My super image in multiple formats!" />
+```
+
+The above code will generate the following:
```

Comment:
```suggestion
The above code will generate the following HTML, and allow the browser to determine the best image to display:
```
I originally came just to add "HTML", but since this will be also material for the blog post draft, I think throwing in a benefit of some kind at the end of the sentence will be helpful for the release blog post AND gives us a bit of a nicer sentence here.

Pick whatever benefit/description you want, but I like a beefier sentence showing off how this is useful here!